src/ResourceManager/Network/Commands.Network/packages.config

```diff
@@ -2,5 +2,8 @@
 <packages>
   <package id="AutoMapper" version="6.0.2" targetFramework="net452" />
   <package id="Microsoft.Azure.KeyVault.Core" version="1.0.0" targetFramework="net45" />
-  <package id="Microsoft.Azure.Management.Network" version="15.1.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Azure.Management.Network" version="17.0.0-preview" targetFramework="net452" />
+  <package id="Microsoft.Rest.ClientRuntime" version="2.3.10" targetFramework="net452" />
```

Comment:
These additional packages should not be here

Reply:
Resolved

Reply:
Please remove these packages again.